doc/nrf/protocols/bt/ble/radio_notification_conn_cb.rst

```diff
@@ -0,0 +1,39 @@
+.. _ug_radio_notification_conn_cb:
+
+Radio Notification callback
+###########################
+
+The Bluetooth Radio Notification connection callback feature is used to inform the application about upcoming radio activity for a Bluetooth connection.
+The :c:member:`bt_radio_notification_conn_cb.prepare` callback is triggered right before a Bluetooth connection event.
+This allows the application to trigger sensor data collection for transmission during the upcoming connection event.
+
+.. figure:: images/ConnEvtCb.svg
+   :alt: Radio Notification callback
+
+   The prepare callback is called before a connection event
+
+The timing of the prepare callback is configured with the parameter ``prepare_time_us``.
+You must set the parameter value large enough to allow the Bluetooth stack to enqueue a data packet for the upcoming connection event.
+
+The :ref:`ble_radio_notification_conn_cb` sample demonstrates how you can use this feature to reduce the latency between data sampling and data transmission.
+
+You can use this feature only with the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` and only when the Bluetooth controller and application are located on the same core.
+
+Radio Notification connection callback with peripheral latency
+**************************************************************
+
+When a connection is configured with peripheral latency, the peripheral device may sleep for ``peripheral_latency`` consecutive connection events to save power.
+The device may wake up more often if, for example, it has data to send.
+The central device assumes that the peripheral will wake up at every connection event.
+
+The :c:member:`ble_radio_notification_conn_cb.prepare` callback will be called before every connection event for both the peripheral and central device.
```

Comment:
Should we combine this with the previous paragraph? I think they logically belong together.